d8s_python/python_data.py

```diff
@@ -359,3 +362,44 @@ def python_type_name(python_type: type) -> str:
 def python_object_type_to_word(python_object: Any) -> str:
     """Convert the given python type to a string."""
     return python_type_name(type(python_object))
+
+
+def _get_importfrom_module_name(node: ImportFrom) -> str:
+    """Extract the module name from an ast.ImportFrom node.
+
+    The module name on the ast.ImportFrom node can be None for relative imports
+    In this case, this function will return the name as the dots from the import statement.
+    A few examples:
+    "from requests import get" -> "get"
```

Comment:
I believe this is incorrect:

```suggestion
    "from requests import get" -> "requests"
```

Reply:
Good catch! I've commited your fix.